api/metrics/v1/metrics.proto

```diff
@@ -36,7 +36,8 @@ message GetMetricsRequest {
 message Metric {
   double value = 1;
   int64 timestamp_ms = 2;
-  repeated string metric_tags = 3;
+  /** metric labels (tags) have a key and a value, https://prometheus.io/docs/concepts/data_model/#notation */
```

Comment:
Idk if this is the best doc to link, as the metric API here could be aggregating from maybe backends beyond just prom